lib/rojo-says.js

```diff
@@ -0,0 +1,72 @@
+const listen = (text) => {
+    const startsWithHello = /hello.*/i;
+    const containsWhen = /when.*/i;
+    const containsWhat = /what.*/i;
+    const containsWhich = /which.*/i;
+    const containsDay = /.+day.*/i;
+    const containsTime = /.+time.*/i;
+    const containsYou = /.+you.*/;
+    const containsHow = /how.*/i;
+    const containsFeeling = /feeling.*/i;
+    const containsProvince = /.+province.*/i;
+    const containsFrom = /.+from.*/i;
+    const containsPhone = /.+phone.*/i;
+    const containsNumber = /.+number.*/i;
+    const containsYour = /.+your.*/i;
+    const containsName = /.*name.*/i;
+    const containsNationality = /.*nationality.*/i;
+    const containsFavourite = /.+favourite.*/i;
+    const containsColor = /.+color.*/i;
+    const containsMonth = /.+month.*/i;
+    const containsSeason = /.+season.*/i;
```

Comment:
This is begging for a refactor. Look at the pattern in your naming, and extract objects:

```js
const startsWith = {
  hello: /hello.*/i,
  ..etc
}

const contains = {
  time: /.+time.*/i,
  ...etc
}
```